doc/rest-api/Administration-backend_swagger.yml

```
@@ -260,6 +260,28 @@ paths:
           description: The messages archived for the given user and the other party.
           schema:
             $ref: '#/definitions/messageList'
+  /stanzas:
+    post:
+      tags:
+        - "One-to-one messages"
+      parameters:
+        - name: message
+          in: body
+          description: The chat stanza
+          required: true
+          schema:
+            title: message
+            type: object
+            properties:
+              stanza:
+                description: the complete stanza
+                type: string
+                required: true
+                example: "<message from=\"alice@wonderlant.lit\" to=\"bob@wonderlant.lit\"><body>whatever</body></message>"
+      description: Send an arbitrary stanza (attributes from and two are required).
```

```suggestion
      description: Send an arbitrary stanza (attributes from and to are required).
```
Or if we can put inline code here, this one would be better
```suggestion
      description: Send an arbitrary stanza (attributes `from` and `to` are required).
```